bugbug/similarity.py

```diff
@@ -280,7 +286,7 @@ def __init__(
             output_prefix="simdata.shdat", corpus=corpus_lsi, num_features=300
         )
 
-    def get_similar_bugs(self, query, k=10):
+    def get_similar_bugs_child(self, query, k=10):
```

Comment:
Let's try to find a nicer name than `get_similar_bugs_child`, maybe `search_similar_bugs`.

Reply:
So `get_similar_bugs` would be the function in the base class, `search_similar_bugs` the function in the children classes.